src/main/java/uk/gov/hmcts/reform/managecase/client/datastore/ChangeOrganisationRequest.java

```diff
@@ -24,9 +30,20 @@
     private final String caseRoleId;
 
     @JsonProperty("RequestTimestamp")
+    @JsonSerialize(using = LocalDateTimeSerializer.class)
     private final LocalDateTime requestTimestamp;
 
     @Setter
     @JsonProperty("ApprovalStatus")
     private String approvalStatus;
+
+    public void validateChangeOrganisationRequest() {
```

Comment:
Minor, but does this need some unit testing?

Reply:
done